WordPress/Classes/Utility/Ratings/AppRatingUtility.h

```diff
@@ -1,15 +1,105 @@
 #import <Foundation/Foundation.h>
 
-// Based loosely on Appirater (https://github.com/arashpayan/appirater)
-
+/**
+ This class will help track whether or not a user should be prompted for an app review. This class is loosely based on Appirater (https://github.com/arashpayan/appirater)
```

Comment:
This is pretty minor, but if you could break comment lines to fit the default size (I think it's 80 chars) it would help for reading them in github.  Documentation generators can deal with hard breaks so it shouldn't be a problem on that front.